packages/google-calendar/src/resources/calendars/calendars.ts

```
@@ -0,0 +1,557 @@
+
+  /**
+   * Deprecated. Please use quotaUser instead.
+   */
+  userIp?: string;
+}
+
+Calendars.ACL = ACL;
+
+export declare namespace Calendars {
+  export {
+    type Calendar as Calendar,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)